pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2482,7 +2479,6 @@
         "netcoreapp2.1": "4.1.0.0",
         "monoandroid10": "Any",
         "monotouch10": "Any",
-        "uap10.0.16300": "4.1.0.0",
```

Comment:
What about all the other libraries that are listed inbox on 16300? Do we need to remove those?

Reply:
I think most (if not all) of the rest of them are just part of the meta package and don't have their own individual package, which is why it isn't really needed for this specific purpose. That said, if you want me to remove them I can do that as well.

Reply:
If they aren't causing any harm don't worry about it.